.obsidian/plugins/heatmap-calendar/styles.css

```
@@ -0,0 +1,173 @@
+}
+.modal.mod-sidebar-layout {
+  width: 700px !important;
+}
+.vertical-tab-content::-webkit-scrollbar {
+  display: none;
+}
+*/
+
+/* ----
+/* ---- DEV Only /END ---- */
+/* ----
```

Stylelint (17.14.0)

[error] Unclosed comment (CssSyntaxError)

(CssSyntaxError)